app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= presenter.attribute_to_html(:date_modified, label: 'Last Modified') %>
```

Comment:
What does this have to do with the rest of the PR?

Reply:
Based on the mockup, this is is moved to the attributes list from it's current location on the show template.

Reply:
👌

Reply:
Can you show me where this is moved from? I'm not seeing it as a `-` in the diff.

Reply:
Ah, should this label be `t('hyrax.base.show.last_modified')` ?

Reply:
This wasn't moved. It was added to attribute_rows per the mock up, it was original just a value in the show page I believe. Can a translate title be used here? If so, I'll make that change later today.